data-prepper-plugins/s3-sink/src/test/java/org/opensearch/dataprepper/plugins/codec/parquet/ParquetOutputCodecTest.java

```diff
@@ -115,11 +117,11 @@ void test_happy_case(final int numberOfRecords) throws Exception {
             parquetOutputCodec.writeEvent(event, outputStream);
         }
         parquetOutputCodec.complete(outputStream);
-        List<Map<String, Object>> actualRecords = createParquetRecordsList(new ByteArrayInputStream(tempFile.toString().getBytes()));
+        List<Map<String, Object>> actualRecords = createParquetRecordsList(new FileInputStream(tempFile));
```

Comment:
We need assertions on all of these to verify the size. Otherwise, the loop below will not assert anything.

```
assertThat(actualRecords.size(), equalTo(inputMaps.size()));
```